lighthouse-core/audits/deprecations.js

```diff
@@ -56,16 +56,34 @@ class Deprecations extends Audit {
   static audit(artifacts) {
     const entries = artifacts.ConsoleMessages;
 
-    const deprecations = entries.filter(log => log.source === 'deprecation')
-    // TODO(M91): Temporary ignore until Chrome M91 became stable version.
-    // M91 doesn't throw deprecation on ::-webkit-details-marker.
-    .filter(log => !log.text.includes('::-webkit-details-marker'))
-    .map(log => {
-      return {
-        value: log.text,
-        source: Audit.makeSourceLocationFromConsoleMessage(log),
-      };
-    });
+    let deprecations;
+
+    if (artifacts.InspectorIssues.deprecations?.length) {
+      deprecations = artifacts.InspectorIssues.deprecations
+        .map(deprecation => {
+          return {
+            value: deprecation.message || '',
+            /** @type {LH.Audit.Details.SourceLocationValue} */
+            source: {
+              type: 'source-location',
+              url: deprecation.sourceCodeLocation.url,
+              urlProvider: 'network',
+              line: deprecation.sourceCodeLocation.lineNumber,
+              column: deprecation.sourceCodeLocation.columnNumber - 1,
+            },
+          };
+        });
+    } else {
```

Comment:
does this need to be an if/else or can these be combined in one go?

Reply:
There are some canary releases where both would be populated.